lua/lazyvim/plugins/extras/util/blame-nvim.lua

```diff
@@ -0,0 +1,9 @@
+return {
+  "FabijanZulj/blame.nvim",
+  config = function()
+    require("blame").setup()
+  end,
```

Comment:
```suggestion
  opts = {},
```
That should be easier for users to change options in their personal config. Otherwise you would have to pass `opts` to the `config` function and also pass them to `setup`.